### PR TITLE
[Trading][Exchanges] Add bybit

### PR DESCRIPTION
**Pull request by valouvaliavlo**

### CODE REVIEW

**Comment by Herklos** [comment]
> _No description provided._

Is it working in spot trading ?

**Comment by valouvaliavlo** [comment]
> > _No description provided._
> 
> Is it working in spot trading ?

About the function overriding: yes, bybit only support 200 candles maxi. This modification is mainly for backtesting download because there is no limit which is specified and this caused an error with this exchange.

The exchange in globality : it seems to work for poeple on discord

File: Trading/Exchange/bybit/bybit_exchange.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        return 'bybit'
+
+    @classmethod
+    def is_supporting_exchange(cls, exchange_candidate_name) -> bool:
+        return cls.get_name() == exchange_candidate_name
+
+    async def get_symbol_prices(self, symbol, time_frame, limit: int = 200, **kwargs: dict):
+        try:
+            candles = await self.connector.client.fetch_ohlcv(symbol, time_frame.value, limit=limit, params=kwargs)
+            return candles
```
Comment (GuillaumeDSM) [inline Trading/Exchange/bybit/bybit_exchange.py:34]: If we do nothing with the candles variable, I think we should return the awaited result directly

**Review by GuillaumeDSM — COMMENTED** [review]
👍

File: Trading/Exchange/bybit/bybit_exchange.py (hunk 1 above)
Comment (valouvaliavlo) [reply Trading/Exchange/bybit/bybit_exchange.py]: exact !

**Review by GuillaumeDSM — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

**Pull request merged**